packages/zeebe-element-templates-json-schema/src/defs/properties/bindingType.json

```diff
@@ -32,6 +32,20 @@
           "value"
         ],
         "properties": {
+          "choices": {
+            "type": "array",
```

Comment:
@barmac 
We should double-check that we are not overwriting anything from the [actual choices](https://github.com/camunda/element-templates-json-schema/blob/0747597d333ff2b88d42b6af90fc6e35d1cebd46/packages/element-templates-json-schema-shared/src/defs/base-properties.json#L127)
I.e, that a choice without a name is suddenly legal. `{"value": "thisHasNoName"}`

Reply:
@barmac Tbh. I can't say for sure, this is nested an anyOf and allOfs.  I'll write a short test case to check this.

Reply:
@barmac I ran a test without `name` in the dropdown items and it correctly threw an error.